base-api.Tests/V1/Infrastructure/ExampleContextTests.cs

```diff
@@ -0,0 +1,24 @@
+using System.Linq;
+using NUnit.Framework;
+using base_api.V1.Domain;
+using UnitTests.V1.Helper;
+
+namespace UnitTests.V1.Infrastructure
+{
+    [TestFixture]
+    public class ExampleContextTest : DbTest
```

Comment:
Again I'm not sure if this makes much sense to test either. It's kind of just testing that the database is set up correctly which again is handled in the gateway tests.  Or is the plan for this to test that the schema that's been imported matches what we have in code?

Reply:
Hmm - that could be the plan, but yes currently we've just got test duplication in here. Will remove this one as our team is really clear on testing strategy so can add whatever they do need; no need for a dummy one here.